web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -268,8 +278,15 @@ def mentor_program_families(self):
             self.program_family, flat=True).distinct()
 
     def location_choices(self):
-        return self.user_query.values(
+        locations = self.user_query.values(
             **_location_lookups(self.location_path)).distinct()
+        remote_location_fields = dict([("location_" + field, F(field))
+                                       for field in LOCATION_FIELDS])
+        remote_location = Location.objects.filter(
+            name="Remote").values(**remote_location_fields).first()
+        locations_list = list(locations)
+        return locations_list.append(
```

Comment:
`append()` does not return a list. `locations_list` should be returned instead

Reply:
can we confirm that 'Remote' is not in the existing `locations_list` before appending?